### PR TITLE
Introducing the validator module

### PR DESCRIPTION
**Pull request by saganatt**
Ciao @ginnocen,
this is the first validation PR - with your skeleton polished. Works ok.

### CODE REVIEW

File: tpcwithdnn/data_loader.py [hunk 1]
```diff
@@ -140,12 +140,10 @@ def get_event_mean_indices(maxrandomfiles, range_mean_index, ranges):
     sel_indices_events_means = random.sample(all_indices_events_means, \
         maxrandomfiles * (range_mean_index[1] + 1 - range_mean_index[0]))
 
-    indices_train = [sel_indices_events_means[index] \
-        for index in range(ranges["train"][0], ranges["train"][1])]
-    indices_test = [sel_indices_events_means[index] \
-        for index in range(ranges["test"][0], ranges["test"][1])]
-    indices_apply = [sel_indices_events_means[index] \
-        for index in range(ranges["apply"][0], ranges["apply"][1])]
+    indices_train = sel_indices_events_means[ranges["train"][0]:ranges["train"][1]]
```
Comment (saganatt) [inline tpcwithdnn/data_loader.py:143]: This code has exactly the same effects, but looks more readable imho

Comment (ginnocen) [reply tpcwithdnn/data_loader.py:143]: Sounds good!

**Comment by ginnocen** [comment]
Thanks @saganatt. Seems fine to me, but I will check with more calm tomorrow. Once this is merged, I would get started trying to add the columns related to the predictions. We can discuss further during tomorrow's meeting.

**Comment by saganatt** [comment]
> Thanks @saganatt. Seems fine to me, but I will check with more calm tomorrow. Once this is merged, I would get started trying to add the columns related to the predictions. We can discuss further during tomorrow's meeting.

ok :-)

**Review by ginnocen — APPROVED** [review]

**Pull request merged**